docs/feature-gates.md

```diff
@@ -250,8 +245,26 @@ metadata:
     ipam.antrea.io/ippools: 'pool1'
 ```
 
+#### Data path change for this feature
+
+When `AntreaIPAM` is enabled, `antrea-agent` will connect the Node's network interface
+to the OVS bridge at startup, and it will detach the interface from the OVS bridge and
+restore its configurations at exit. Node may lose network connection when `antrea-agent`
+or OVS daemons are stopped unexpectedly, which can be recovered by rebooting the Node.
+`AntreaIPAM` Pods' traffic will not be routed by local Node's network stack.
+
+All traffic to a local Pod will be sent to the Pod's OVS port directly, after the
+destination MAC is rewritten to the Pod's MAC address, including `AntreaIPAM` Pods and
+regular `Subnet per Node` IPAM Pods, even they are not in the same subnets. Inter-Node
+traffic will be sent from source Node to destination Node directly.
```

Comment:
Better to add more details here, like: "Inter-Node traffic will be sent to the Node network from the source Node, and forwarded to the destination Node by the Node network."

Reply:
Fixed here.
I also planned to add a matrix in future release to explain the data path for every endpoint, e.g. Per-Node IPAM Pod, AntreaIPAM Pod default VLAN, AntreaIPAM Pod VLAN <A>, AntreaIPAM Pod VLAN <B>, NodeIP, Service ClusterIP, externalIP